controllers/users.controller.js

```diff
@@ -0,0 +1,9 @@
+const fetchUsers = require("../models/users.model");
+
+const getUsers = (req, res) => {
+  fetchUsers().then((Users) => {
```

Comment:
users shouldn't be capitalised - I'll approve now but please change before your next ticket!